src/app/services/player.service.js

```diff
@@ -9,6 +9,9 @@ class PlayerService {
     this.uiSettingsService = uiSettingsService;
     this.$document = $document;
 
+    this.serverStateNumeral = -1;
```

Comment:
use self explanatory var names: serverStateNumeral and stateNumeral aren't clear